app/models/user.rb

```diff
@@ -0,0 +1,13 @@
+class User < ActiveRecord::Base
+  has_many :people
+  has_many :companies
+
+  def self.find_or_create_by_auth(auth_data)
+    user = User.where(provider: auth_data['provider'], uid: auth_data['uid']).first_or_create
```

Comment:
Line is too long. [93/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.